Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -259,5 +260,50 @@ protected void CheckMixedRealityInputActions()
                 }
             }
         }
+
+        /// <summary>
+        /// Applies the most relevant settings of DataProviderConfigurations
+        /// </summary>
+        /// <param name="typeName">name of the Type of the Data Provider</param>
+        /// <param name="providerAttribute"> the corresponding attribute</param>
+        /// <param name="providerName"> the target provider Name</param>
+        /// <param name="configurationProfile">the target configuration profile</param>
+        /// <param name="runtimePlatform">the target runtime plaform</param>
+        protected void ApplyDataProviderConfiguration(
+            string typeName,
+            MixedRealityDataProviderAttribute providerAttribute,
+            SerializedProperty providerName,
+            SerializedProperty configurationProfile,
+            SerializedProperty runtimePlatform)
+        {
+            if (typeName != null)
+            {
+                if (providerAttribute != null)
+                {
+                    providerName.stringValue = !string.IsNullOrWhiteSpace(providerAttribute.Name) ? providerAttribute.Name : typeName;
+                    configurationProfile.objectReferenceValue = providerAttribute.DefaultProfile;
+                    runtimePlatform.intValue = (int)providerAttribute.RuntimePlatforms;
+                }
+                else
+                {
+                    providerName.stringValue = typeName;
+                }
+
+                serializedObject.ApplyModifiedProperties();
+            }
+        }
+
+        /// <summary>
+        /// Returns the MixedRealityDataProviderAttribute on a given type
+        /// </summary>
+        /// <param name="type">The type on which to look after the attribute</param>
+        /// <returns>The MixedRealityDataProviderAttribute for the given type</returns>
+        protected MixedRealityDataProviderAttribute GetTypeNameAndProvider(SystemType type)
```

Comment:
This seems incorrectly / misleadingly named.

In the original PR, @Troy-Ferrell asked for this to be removed: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6133/files#r332631565

Is it used?

Reply:
it's been removed. it was not used